app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,73 @@
+package com.codesoom.assignment.application;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.exception.ProductNonExistException;
+import com.codesoom.assignment.repository.ProductRepository;
+
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 장난감을 생성하고 반환하는 메서드
+     * @param product
+     * @return 장난감
+     */
```

Comment:
메서드를 설명하는 말에 메서드라고 작성하지 않아도 됩니다.
```suggestion
    /**
     * 장난감을 생성해 리턴합니다.
     * @param product
     * @return 장난감
     */
```

Reply:
반환하는 것을 명시하는 것이 확실히 가독성 면에서 나아보입니다. 그리고 주석이든 문서든 누구나 아는 사실을 명시하는 것을 주의해야할 것 같습니다.